model/RevisionStorageInterface.php

```diff
@@ -71,6 +71,10 @@ public function getData(Revision $revision);
      */
     public function getResourcesUriByQuery(string $query, array $options = [], string $predicate = OntologyRdfs::RDFS_LABEL);
 
+    /**
+     * @return array
+     */
+    public function getResourcesDataByQuery(string $query, array $options = [], string $predicate = OntologyRdfs::RDFS_LABEL): array;
```

Comment:
```suggestion
    public function getResourcesDataByQuery(string $query, array $options = [], string $predicate = OntologyRdfs::RDFS_LABEL): array;
```

as per squad conventions, we may omit phpdoc comments, which are not providing value on top of defined strong typing